apps/project/schema.py

```diff
@@ -612,3 +637,7 @@ def resolve_public_projects(root, info, **kwargs) -> QuerySet:
     @staticmethod
     def resolve_public_projects_by_region(*args, **kwargs):
         return Query.resolve_projects_by_region(*args, **kwargs)
+
+    @staticmethod
+    def resolve_project_stat_summary(root, info, **kwargs):
+        return info.context.dl.project.resolve_project_stat_summary()
```

Comment:
We don't need data loader here, as they are used when we need to do merge N times function call to 1.
For this, moving the fetch logic to the first section of the file is okay. Or define the function as regular and import it here directly from dataloder.